docs/pages/api/snackbar-content.md

```diff
@@ -27,6 +27,7 @@ You can learn more about the difference by [reading this guide](/guides/minimizi
 | <span class="prop-name">action</span> | <span class="prop-type">node</span> |  | The action to display. |
 | <span class="prop-name">classes</span> | <span class="prop-type">object</span> |  | Override or extend the styles applied to the component. See [CSS API](#css) below for more details. |
 | <span class="prop-name">message</span> | <span class="prop-type">node</span> |  | The message to display. |
+| <span class="prop-name">role</span> | <span class="prop-type">'alert'<br>&#124;&nbsp;'alertdialog'</span> | <span class="prop-default">'alert'</span> | The role of the SnackbarContent. If the SnackbarContent includes interactive controls, such as a link, "alertdialog" should be used; otherwise use "alert". |
```

Comment:
```suggestion
| <span class="prop-name">role</span> | <span class="prop-type">'alert'<br>&#124;&nbsp;'alertdialog'</span> | <span class="prop-default">'alert'</span> | The role of the SnackbarContent. If the SnackBar requires focus to be closed the `alertdialog` role should be used instead. |
```

>  If an alert requires focus to close the alert, then content authors SHOULD use alertdialog instead.

-- https://www.w3.org/TR/wai-aria-1.2/#alert

And then we can drop it from every demo that doesn't require a user action to be closed. I prefer this anyway. The less aria attributes are required for devs the better.

`alertdialog` would also require an accessible name via e.g. `aria-labelledby`. I don't think Snackbars should ever be used as an alert dialog.

Reply:
Makes sense! I can update the examples and the docs.